fem/gslib.cpp

```diff
@@ -60,51 +61,40 @@ void FindPointsGSLIB::Setup(Mesh &m, double bb_t, double newt_tol, int npt_max)
    MFEM_VERIFY(m.GetNodes() != NULL, "Mesh nodes are required.");
 
    mesh = &m;
-   const GridFunction *nodes = mesh->GetNodes();
-   const FiniteElementSpace *fes = nodes->FESpace();
-
-   dim = mesh->Dimension();
-   const int NE = mesh->GetNE(),
-             dof_cnt = fes->GetFE(0)->GetDof(),
-             pts_cnt = NE * dof_cnt;
-   gsl_mesh.SetSize(dim * pts_cnt);
+   dim  = mesh->Dimension();
+   unsigned dof1D = mesh->GetNodalFESpace()->GetFE(0)->GetOrder() + 1;
```

Comment:
Could we factor out `mesh->GetNodalFESpace()->GetFE(0)` (here and below) as `FiniteElement *fe = ...`, please